pontoon/base/models.py

```diff
@@ -192,19 +203,36 @@ def user_managed_locales(self):
     return [locale.code for locale in locales]
 
 
-@property
-def user_rights(self):
+def user_role(self, managers=None, translators=None):
```

Comment:
Could you add a comment that explains performance issues and why cases user_roles() helps to minimize number of queries?